hyperstart.go

```diff
@@ -206,9 +206,11 @@ func (h *hyper) buildNetworkInterfacesAndRoutes(pod Pod) ([]hyperstart.NetworkIf
 			MACAddr:     endpoint.GetHardwareAddr(),
 		}
 
-		switch endpoint.(type) {
+		switch ep := endpoint.(type) {
 		case *VirtualEndpoint:
 			iface.MTU = netIface.MTU
+		case *PhysicalEndpoint:
```

Comment:
@amshinde why do you not use an GetMTU() interface here. Then you do not need to switch on the concrete type.

Reply:
Also all the more reason for https://github.com/containers/virtcontainers/pull/470